processor/metricstransformprocessor/operation_aggregate_labels.go

```diff
@@ -248,7 +248,6 @@ func mergeHistogramDataPoints(dpsMap map[string]pmetric.HistogramDataPointSlice,
 			if dp.HasMax() && dp.Max() < dps.At(i).Max() {
 				dp.SetMax(dps.At(i).Max())
 			}
-			dps.At(i).Exemplars().MoveAndAppendTo(dp.Exemplars())
```

Comment:
Is this an intentional removal?

Reply:
yes, because 3 lines below we do it one more time. Do you think it is not a bug?

Reply:
Oh right. thanks for fixing it :)